pkg/machine/hyperv/hutil/hutil.go

```diff
@@ -0,0 +1,35 @@
+package hutil
```

Comment:
Is the new package required for circular deps? or you preferred to do it this way?

Reply:
yes, circular deps. So i copied what they did on wsl